README.md

```diff
@@ -236,9 +236,9 @@ typedef enum : NSUInteger {
 
 + (NSValueTransformer *)updatedAtJSONTransformer {
     return [MTLValueTransformer reversibleTransformerWithForwardBlock:^(NSString *str) {
-        return [self.dateFormatter dateFromString:str];
+        return [self.class.dateFormatter dateFromString:str];
```

Comment:
but in first demo is `self.class.dateFormatter`
or don't declare in the class method?
`self` can't call the class method